bucket/portwarden.json

```diff
@@ -0,0 +1,38 @@
+{
+    "version": "1.0.0",
+    "description": "Create Encrypted Backups of Your Bitwarden Vault with Attachments",
+    "homepage": "https://github.com/vwxyzjn/portwarden",
+    "license": "MIT",
+    "architecture": {
+        "32bit": {
+            "url": "https://github.com/vwxyzjn/portwarden/releases/download/1.0.0/portwarden_windows_386.exe",
+            "hash": "358e9591939cf69600a7ed0cd6ddd9951abdff9bf728635326fd293d4828c4cd"
+        },
+        "64bit": {
+            "url": "https://github.com/vwxyzjn/portwarden/releases/download/1.0.0/portwarden_windows_amd64.exe",
+            "hash": "8477ccab21716a6c3f89302dd44756ae05181d9ec34ccd3aad22a9b43bf79ef6"
+        }
+    },
+    "depends": [
+        "bitwarden-cli"
+    ],
+    "bin": [
+        [
+            "portwarden_windows_*.exe",
+            "portwarden"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/vwxyzjn/portwarden"
+    },
```

Comment:
```suggestion
```

The package hasn't received updates in a while, so I will recommend removing checkver for now.